stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/country-site-header.scss

```diff
@@ -46,6 +46,11 @@
         display: flex;
         margin-top: 0.3rem;
       }
+
+      a {
+        font-size: inherit !important;
```

Comment:
Hi @Jura, yeah I'm not a fan of !important's either. Let me see if I can work it a little more and avoid it. I've already reworked some other tasks and removed the !important flags from them.

Reply:
@Jura this is updated. Ok to resolve?

Reply:
Sure, the mechanics here is I'm raising concern, if you agree and submitted fix - just resolve conversation. If there is a regression associated with that fix - I'll add new comment, otherwise we will have this review resolved